module6/interviews/questions-4545.md

```diff
@@ -0,0 +1,71 @@
+---
+layout: page
+title: Interview Questions
+---
+
+You can ask these questions in any order.  There are some questions each week that we know students will not be able to answer - this is by design to see how they exit a tough situation.  Keep track on notes on [this tracker](https://docs.google.com/spreadsheets/d/1eMtwW-06Wv0YAdAnZVm5k8_VFNKEKzQeq24kGxMTHbQ/edit?usp=sharing), please!
+
+
+## Week 1
+* Describe the 4 Pillars of OOP
+* What is HTTP and give a brief definition?
+* Explain Routing in .NET Core MVC?
+* What happens if the inherited interfaces have conflicting method names?
```

Comment:
The wording of this question feels a bit off to me. I have only used inherited for classes, and then use implemented for interfaces.

Maybe we reword to "What happens if a class implements two interfaces that have conflicting method names?"

Reply:
I updated the wording, and left it a bit confusing still.  This is one that I think will be difficult for students and will require them to ask questions of the interviewer to understand what is being asked. 🤷‍♀️